src/main/java/io/spring/githubchangeloggenerator/ChangelogGenerator.java

```diff
@@ -159,8 +164,26 @@ private String getLinkToIssue(Issue issue) {
 		if (this.excludeContributors.contains("*")) {
 			return Collections.emptySet();
 		}
-		return issues.stream().filter((issue) -> issue.getPullRequest() != null).map(Issue::getUser)
-				.filter(this::isIncludedContributor).collect(Collectors.toSet());
+		return issues.stream().map(this::getPortedReferenceIssue).filter((issue) -> issue.getPullRequest() != null)
+				.map(Issue::getUser).filter(this::isIncludedContributor).collect(Collectors.toSet());
+	}
+
+	private Issue getPortedReferenceIssue(Issue issue) {
```

Comment:
@scottfrederick I don't think this would work for an issue that is a forward-port of a forward-port, would it? For example, if the original issue is in `2.2.x` and we're generating release notes for `2.4.x`.